docs/_includes/toggle-button.html

```diff
@@ -0,0 +1,492 @@
+<div id="toggle-button">
+    {% include section-header.html name="toggle-button" version=page.versions.toggle-button %}
+
+    <p>The Toggle Button is a button that can be toggled/pressed on or off. It is an atomic piece of the Toggle Button Group.</p>
+    
+    <p>Titles are required and other elements, such as subtitles and images can be included as well.</p>
```

Comment:
We can probably remove this line now from here, as it might cause some confusion now that we are showing minimal view first.

```suggestion
```

Reply:
Aside from this change request, I believe I've addressed all others.

Reply:
> Doesn't the rest of the text simply imply that more options are ahead

It doesn't to me. I would introduce things as they come in, which the list view section already does with its heading structure.